kitsune/questions/config.py

```diff
@@ -724,6 +724,52 @@
                 ),
             },
         ),
+        (
+            "hubs",
+            {
+                "name": _lazy("Hubs"),
+                "subtitle": _lazy("Mozilla Hubs support"),
```

Comment:
should this be "A social 3D platform for collaborating with friends, family and colleagues" (the description in staging)?

Reply:
This is the reason I have this as draft. I am waiting for confirmation from Joni